std/uni.d

```diff
@@ -8324,15 +8332,15 @@ auto asUpperCase(Range)(Range str)
     assert("hEllo".asUpperCase.equal("HELLO"));
 }
 
-// explicitly undocumented
+/// ditto
 auto asLowerCase(Range)(auto ref Range str)
```

Comment:
IMO the ddoc looks much better then the ddox version on functions with multiple overloads